### PR TITLE
Add molecule tests

### PR DESCRIPTION
**Pull request by andykrohg**
Currently simply creates a `DevSecOpsWorkshop` CR and verifies that the operator reports it as **Successful**. We'll need better namespace encapsulation to allow us to test both `DevSecOpsWorkshop` and `TsscPlatform` CRs in the same test cluster since they deploy conflicting flavors of tools to the `devsecops` namespace.

In lieu of loading the assets into an ephemeral Kind cluster, we're now deploying this to an existing OCP cluster by way of the environment variables in Travis:
* `OCP_SERVER`
* `OCP_USERNAME`
* `OCP_PASSWORD`

In lieu of loading the operator image into the internal registry (as in `kind load docker-image`), we do a push/pull to `quay.io/redhatgov/tssc-operator:test`.

### CODE REVIEW

**Review by roller1187 — APPROVED** [review]
Nice work!

File: molecule/ocp-cluster/molecule.yml [hunk 1]
```diff
@@ -21,15 +21,17 @@ provisioner:
...
       localhost:
         ansible_python_interpreter: '{{ ansible_playbook_python }}'
         config_dir: ${MOLECULE_PROJECT_DIRECTORY}/config
         samples_dir: ${MOLECULE_PROJECT_DIRECTORY}/config/samples
         project_dir: ${MOLECULE_PROJECT_DIRECTORY}
-        operator_image: testing-operator
-        operator_pull_policy: "Never"
+        quay_username: "{{ lookup('env', 'QUAY_USERNAME') }}"
+        quay_password: "{{ lookup('env', 'QUAY_PASSWORD') }}"
+        operator_image: quay.io/redhatgov/tssc-operator:test
```
Comment (roller1187) [inline molecule/ocp-cluster/molecule.yml:33]: Don't we need to also add a place in the config/manage/kustomize.yaml to push the image to quay.io using "latest"?

Comment (andykrohg) [reply molecule/ocp-cluster/molecule.yml:33]: Negative- that's performed separately after molecule tests by `hack/operate.sh`

**Comment by jflowers** [comment]
lgtm

**Review by jflowers — APPROVED** [review]

**Pull request merged**